monster_waves/RandomGenerator.hpp

```diff
@@ -0,0 +1,13 @@
+#include <random>
+
+
+class RandomNumber
+{
+public:
+    RandomNumber();
+
+    template<typename T>
```

Comment:
@JakubMastalski wszystko jest ok. Przypominam tylko o formatowaniu. Jeśli nie dodajesz spacji po nawiasach to rób to wszędzie, a nie tylko tu. To sugestia, zrobisz z tym co chcesz. Po za tym plik wygląda ok.